GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -3262,6 +3263,13 @@ private void interactiveAddToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void Amend_CheckedChanged(object sender, EventArgs e)
         {
+            ResetAuthor.Enabled = Amend.Checked;
```

Comment:
I would not show it at all to unclutter the UI.

Reply:
Will change it.